hexadecimal/hex.py

```diff
@@ -0,0 +1,87 @@
+import sys
```

Comment:
* Invalid PULL Commits: *

** Invalid Commit Message: HEX-1022 Added hex.py **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']

** Invalid Commit Message: HEX-1022 Added hex_tests.py **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']

** Invalid Commit Message: HEX-1022 Added TestHexNUmber **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']

** Invalid Commit Message: HEX-1022 Refactored TestHexNUmber **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']

** Invalid Commit Message: HEX-1022 Added TestSolution **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']

** Invalid Commit Message: HEX-1022 Refactored hex.py **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']

** Invalid Commit Message: HEX-1022 Added some test in TestSolution **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']

** Invalid Commit Message: HEX-1022 Refactored code **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']